bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/config/ZWaveConfiguration.java

```diff
@@ -353,6 +355,18 @@ else if(node.getSendCount() > 20 && (node.getRetryCount() * 100 / node.getSendCo
 				record.value = node.getLocation();
 				records.add(record);
 
+				ZWaveSwitchAllCommandClass switchAllCommandClass = (ZWaveSwitchAllCommandClass)node.getCommandClass(ZWaveCommandClass.CommandClass.SWITCH_ALL);
```

Comment:
I don't think putting this here is a good move - I think either in it's own group as you have commented out below, or in the config group. I'm thinking ahead to OH2 here...